packages/lit-html/src/directive.ts

```diff
@@ -12,9 +12,17 @@ import {
   ElementPart,
 } from './lit-html';
 
-export type DirectiveClass = {
+export {
+  AttributePart,
+  BooleanAttributePart,
+  ChildPart,
+  EventPart,
+  Part,
```

Comment:
I think you need to add `ElementPart`

Reply:
I didn't export it because it doesn't exist in Lit1. But that makes sense to do here